discovery-infra/tests/test_kube_api.py

```diff
@@ -184,6 +207,22 @@ def test_kube_api_late_binding_ipv4_single_node(self, unbound_single_node_cluste
 
         self._late_binding_install(cluster_deployment, agent_cluster_install, agents, nodes, is_ipv4=True)
 
+    @JunitTestSuite()
+    @pytest.mark.kube_api
+    def test_kube_api_late_unbinding_ipv4_single_node(
+        self, unbound_single_node_cluster_hold_installation, unbound_single_node_infraenv
+    ):
+        infra_env, nodes = unbound_single_node_infraenv
+        cluster_deployment, agent_cluster_install, cluster_config = unbound_single_node_cluster_hold_installation
+
+        agents = infra_env.wait_for_agents(len(nodes))
+        self._late_binding_install(
+            cluster_deployment, agent_cluster_install, agents, nodes, is_ipv4=True, hold_install=True
+        )
+
```

Comment:
Maybe need to check that host is bound? `wait_for_agents_to_be_bound`

Reply:
fixed